fun-packs/android-appcompat-with-views-dsl/build.gradle.kts

```diff
@@ -11,7 +11,7 @@ plugins {
 }
 
 android {
-    setDefaults()
+    setDefaults(project)
```

Comment:
After my suggestion above gets accepted, all changes like this one can be reverted :)